### PR TITLE
chore: Upgrade to Quay image 3.7.0 (PROJQUAY-3867)

### PR DESCRIPTION
**Pull request by bcaton85**
Upgrading the Quay image to use 3.7.0.

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -253,6 +253,7 @@ func getImageMetadata(app, imageName, archivePath string) string {
 					--change 'ENV LANG=en_US.utf8' \
 					--change 'ENV QUAYDIR=/quay-registry' \
 					--change 'ENV QUAYCONF=/quay-registry/conf' \
+					--change 'ENV QUAYRUN=/quay-registry/conf' \
```
Comment (jonathankingfc) [inline cmd/utils.go:256]: Why is this line needed?

Comment (bcaton85) [reply cmd/utils.go:256]: Quay will fail to startup. It's referenced in a systemd template and if it isn't set the templating fails. I believe it was recently introduced.

**Comment by HammerMeetNail** [comment]
Removing `community.crypto` looks to resolve the install issue, not sure why CI is failing. I'm able to build and install on a test VM using this branch. Retriggering CI and watching VMs....

**Comment by HammerMeetNail** [comment]
Seeing this error in the `quay-app` logs on the remote offline host:

```
Running init script '/quay-registry/conf/init/supervisord_conf_create.sh'
Running init script '/quay-registry/conf/init/zz_boot.sh'
/quay-registry/data/secscan_model/__init__.py:28: DeprecationWarning: Call to deprecated class V2SecurityScanner. (Will be replaced by a V4 API security scanner soon)
  self._legacy_model = V2SecurityScanner(app, instance_keys, storage)
Failed to validate security scanner V2 configuration
/quay-registry/data/secscan_model/__init__.py:30: DeprecationWarning: Call to deprecated class NoopV2SecurityScanner. (Will be replaced by a V4 API security scanner soon)
  self._legacy_model = NoopV2SecurityScanner()
Generated new service key ZpqCzX8CPvzPuRmEjfa0-hEfaZiWIljqsNOc4J-ds70
Error: Format string '%(ENV_QUAYRUN)s/supervisord.sock' for 'unix_http_server.file' contains names ('ENV_QUAYRUN') which cannot be expanded. Available names: ENV_CONFIG_APP_PASSWORD, ENV_DB_CONNECTION_POOLING_REGISTRY, ENV_HOME, ENV_HOSTNAME, ENV_LANG, ENV_LC_CTYPE, ENV_OPERATOR_ENDPOINT, ENV_PATH, ENV_PWD, ENV_PYTHONIOENCODING, ENV_PYTHONUNBUFFERED, ENV_PYTHON_ROOT, ENV_PYTHON_VERSION, ENV_QUAYCONF, ENV_QUAYDIR, ENV_QUAYPATH, ENV_QUAY_CONFIG_READ_ONLY_FIELD_GROUPS, ENV_RED_HAT_QUAY, ENV_SHLVL, ENV_TERM, ENV_container, here
For help, use /usr/local/bin/supervisord -h
```

**Comment by bcaton85** [comment]
That should have been resolved by adding the `QUAYRUN` env var to [this line.](https://github.com/quay/mirror-registry/pull/65/files#diff-5b14f474a50427ede31c39509d3921643c7f757a8c680f3a8bdf8c072e1c2160R256) Is it missing somewhere else? That's the only place i had to add it for my tests.

**Comment by HammerMeetNail** [comment]
> That should have been resolved by adding the `QUAYRUN` env var to [this line.](https://github.com/quay/mirror-registry/pull/65/files#diff-5b14f474a50427ede31c39509d3921643c7f757a8c680f3a8bdf8c072e1c2160R256) Is it missing somewhere else? That's the only place i had to add it for my tests.

Maybe here: https://github.com/quay/mirror-registry/blob/15b55652f106bab4cf1dd8a7c2305c0bc6955b3f/ansible-runner/context/app/project/roles/mirror_appliance/tasks/autodetect-image-archive.yaml#L29

Just pushed up a change and triggered CI

**Review by HammerMeetNail — APPROVED** [review]
LGTM 👍
